types/graphql-resolvers/index.d.ts

```diff
@@ -0,0 +1,348 @@
+// Type definitions for graphql-resolvers 0.2
+// Project: https://github.com/lucasconstantino/graphql-resolvers#readme
+// Definitions by: Mike Engel <https://github.com/mike-engel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { IFieldResolver } from "graphql-tools";
+
+// tslint:disable-next-line:export-just-namespace
+export = GraphQLResolvers;
+export as namespace GraphQLResolvers;
+
+declare namespace GraphQLResolvers {
+    const skip: undefined;
+
+    interface TArgs {
+        [argument: string]: any;
+    }
+
+    function combineResolvers<TSource = any, TContext = any, TA1 = TArgs>(
+        r1: IFieldResolver<TSource, TContext, TA1>
+    ): IFieldResolver<TSource, TContext, TArgs>;
+    function combineResolvers<
+        TSource = any,
+        TContext = any,
+        TA1 = TArgs,
```

Comment:
`TA1` is only used once -- Can't this whole series of functions just be `function combineResolvers<TSource = any, TContext = any>(...resolvers: IFieldResolver<TSource, TContext, any>[]): IFieldResolver<TSource, TContext, TArgs>;` ?

Reply:
That's how I originally had it, but ended up splitting all of these out so that the third argument, `TArgs` from the `IFieldResolver` type could be specified if needed. This is similar to how packages like `ramda` and `lodash` type non-fixed arity functions like `compose`, `pipe`, etc. All `TA` args have a default of `TArgs` in case users *don't* want to manually type all the args.

I'll take any suggestions or compromises though :)